client/my-sites/promote-post-i2/components/campaign-item-details/style.scss

```diff
@@ -339,6 +339,10 @@ body.is-section-promote-post-i2 .layout__content .campaign-item-details.main {
 
 		.campaign-item-details__impressions .campaign-item-details__main-stats-row {
 			border-bottom: 1px solid #ddd;
+
+			&:only-child {
+				border-bottom: none;
```

Comment:
Hides a double border when stats chart was not present.
![Screenshot 2024-11-22 at 13 01 03](https://github.com/user-attachments/assets/9bc53896-a971-457a-ba73-5e28569d6510)